WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsRow.swift

```diff
@@ -27,16 +28,22 @@ struct WooShippingCustoms: View {
                 .padding(.horizontal, 10)
 
             PencilEditButton {
-                // TODO: Add action
+                showCustomsForm.toggle()
             }
             .accessibilityLabel(Text(Localization.editButtonAccessibilityLabel))
         }
         .padding(Layout.borderPadding)
         .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderWidth)
+        .sheet(isPresented: $showCustomsForm) {
+            WooShippingCustomsForm(viewModel: WooShippingCustomsFormViewModel(internationalTransactionNumber: "123",
+                                                                              returnToSenderIfNotDelivered: true))
+
```

Comment:
nit: extra new line

Reply:
Thanks! Removed in 974cce396fb6e487045a6ab08cb21cc0a1aa80ea